src/components/Footer/Footer.js

```diff
@@ -35,9 +35,9 @@ const Footer = () => {
           <div className="footer__contact--col">Mon-Fri, 8:30 am - 5 pm Saturday & Sunday, Closed</div>
         </div>
         <div className="footer__social">
-          <a href="https://twitter.com/" target="_blank" rel="noopener noreferrer"><Twitter className='footer__social-icon' /></a>
-          <a href="https://instagram.com/" target="_blank" rel="noopener noreferrer"><Instagram className='footer__social-icon' /></a>
-          <a href="https://facebook.com/" target="_blank" rel="noopener noreferrer"><Facebook className='footer__social-icon' /></a>
+          <a href="https://www.youtube.com/c/HawaiiUnitedOkinawaAssociation" target="_blank" rel="noopener noreferrer"><YouTube className='footer__social-icon' /></a>
```

Comment:
Youtube component should be on its own line.

```suggestion
          <a href="https://www.youtube.com/c/HawaiiUnitedOkinawaAssociation" target="_blank" rel="noopener noreferrer">
            <YouTube className='footer__social-icon' />
          </a>
```